### ISSUE TITLE
reproducibility for HalfCheetah-v2

### ISSUE THREAD

**Issue by tldoan** [pranz24/pytorch-soft-actor-critic#4]
Hi,

I ran your code by just setting timestep to 3 millions like in the official paper (the other parameters were let by default like in your code). I couldn't reproduce the 15,000 result of the paper. (Joined the plot over 4 seeds).
Is there any specific parameters to set  ?

Thank you !
![halfcheetah_sac](https://user-images.githubusercontent.com/8826872/52079917-b9614000-253a-11e9-8d31-18de5a715dc6.png)

**Comment by pranz24** [comment]
Most of the parameters that I used are taken directly from the paper. I haven't tried any other set of parameters for the reproducing the 15,000 result given in the paper.
You can refer to the parameters used in other implementations of SAC(pytorch):
1. [rlkit](https://github.com/vitchyr/rlkit)
2. [firedup](https://github.com/kashif/firedup)
3. [Horizon](https://github.com/facebookresearch/Horizon)

**Comment by tldoan** [comment]
Hi,

but you DO confirm me that with your actual code you could reproduce the 15,000 result??

**Comment by pranz24** [comment]
I couldn't, although I never ran the code for more than 1 million timesteps.
The best I got was around 8,200 using `--automatic_entropy_tuning` as `True`. (According to the paper it should be around 10,000.)
Maybe you can reproduce the results, given in the paper, using some other parameters.

**Comment by tldoan** [comment]
Hi,

I actually set automatic_entropy_tuning to True too and I got 10,000 as a final results which is far from the 15,000. (although it is just 1 seed )  
Thank you for having coded the soft actor critic in pytorch version, much easier to handle than tensorflow.
Will try to look into the other implemention you refered to.
![image 2](https://user-images.githubusercontent.com/8826872/52196562-60c2c900-2800-11e9-85ec-6bcddb1b94d6.png)

**Comment by toshikwa** [comment]
HI, I tried to reproduce results of the paper and I modified the code in this repo a little.
The result is better than yours, I'm not sure it's due to my modifications.(I will test it later and send PR.)
Below is the result of HalfCheetah-v2

![test_rewards](https://user-images.githubusercontent.com/37267851/60399421-e9bb1200-9b9e-11e9-9f60-bbbfd0a395a1.png)

Anyway, thank you for sharing your codes!!
